tools/reviewer/local_server/web_login_js/README.md

```diff
@@ -0,0 +1,14 @@
+# Firestore web login for StartupOS local server
+
+### Setting
+Port and config: `./src/config.js`
+
+### Development
+```
+npm start
```

Comment:
If this is just plain HTML+js, why do we need npm? Please document.

Reply:
There was a small development `expressjs` server, which launched the HTML+js code locally.
`expressjs` require npm.
But we can run localserver from outside, it doesn't have to be part of the project. So I removed it.